blogposts/2021/release-3.30.md

```diff
@@ -0,0 +1,204 @@
+    category: Search
+  - description: "Experimental: Search-based code insights can run over all repositories on the instance. To enable, use the feature flag `\"experimentalFeatures\": { \"codeInsightsAllRepos\": true }` and tick the checkbox in the insight creation/edit UI."
+    url: https://github.com/sourcegraph/sourcegraph/issues/22759
+    category: Search
+  - description: Search References is a new search sidebar section to simplify learning about the available search filters directly where they are used.
+    url: https://github.com/sourcegraph/sourcegraph/issues/21539
+    category: Search
+# Changed
+  - description: Backend Code Insights only fills historical data frames that have changed to reduce the number of searches required.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22298
+    category: Search
+  - description: Backend Code Insights displays data points for a fixed 6 months period in 2 week intervals, and will carry observations forward that are missing.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22298
+    category:
+  - description: Backend Code Insights now aggregate over 26 weeks instead of 6 months.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22527
+    category:
+  - description: "Search queries now disallow specifying `rev:` without `repo:`. Note that to search across potentially multiple revisions, a query like `repo:.* rev:\u003crevision\u003e` remains valid."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22705
+    category: Search
+  - description: The extensions status bar on diff pages has been redesigned and now shows information for both the base and head commits.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22123/files
+    category:
+  - description: "The `applyBatchChange` and `createBatchChange` mutations now accept an optional `publicationStates` argument to set the publication state of specific changesets within the batch change. [#22485](https://github.com/sourcegraph/sourcegraph/pull/22485) and"
+    url: https://github.com/sourcegraph/sourcegraph/pull/22854
+    category: Batch Changes
+  - description: Search queries now return up to 80 suggested filters. Previously we returned up to 24.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22863
+    category: Search
+  - description: "GitHub code host connections can now include `repositoryQuery` entries that match more than 1000 repositories from the GitHub search API without requiring the previously document work-around of splitting the query up with `created:` qualifiers, which is now done automatically."
+    url: https://github.com/sourcegraph/sourcegraph/issues/2562
+    category: Search
+# Fixed
+  - description: The Batch Changes user and site credential encryption migrators added in Sourcegraph 3.28 could report zero progress when encryption was disabled, even though they had nothing to do. This has been fixed, and progress will now be correctly reported.
```

Comment:
@christinaforney (since Malo is on PTO) - is this update worth including?

Reply:
Probably not, let's pull.